canonical/maas-ui
# chore: upgrade react components

## Done

- chore: upgrade react components
- update tests to hover a tooltip trigger before assertions
- add `expectTooltipOnHover` test util to reduce repetition
- fix react type resolution issues
- fix type errors

## QA 
### QA Steps
- Go to machine list and hover over any icons that trigger a tooltip to verify it works correctly
- Navigate to a few other pages ensuring there's nothing unexpected
